railties/lib/rails/application/configuration.rb

```diff
@@ -215,6 +215,7 @@ def load_defaults(target_version)
             active_support.key_generator_hash_digest_class = OpenSSL::Digest::SHA256
             active_support.remove_deprecated_time_with_zone_name = true
             active_support.cache_format_version = 7.0
+            active_support.fallback_to_marshal_serialization = false
```

Comment:
These configuration values have tests in `railties/test/application/configuration_test.rb`. Probably good that we add new tests for our new config values.
https://github.com/fresh-eggs/rails/blob/344095d6ab8d0516fb4293dfa511945123fc5d91/railties/test/application/configuration_test.rb#L2348-L2358